src/main/java/org/alliancegenome/curation_api/dao/AGMDiseaseAnnotationDAO.java

```diff
@@ -12,4 +17,15 @@ protected AGMDiseaseAnnotationDAO() {
 		super(AGMDiseaseAnnotation.class);
 	}
 
+	public Set<String> getGeneExpressionMap() {
```

Comment:
should this have a different name? getAnnotationMap or getAnnotationSet or something?